user-interface/src/case-assignment/AssignAttorneyModal.test.tsx

```diff
@@ -1,61 +1,66 @@
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
-import AssignAttorneyModal, { AssignAttorneyModalRefType } from './AssignAttorneyModal';
+import AssignAttorneyModal, {
+  AssignAttorneyModalProps,
+  AssignAttorneyModalRefType,
+} from './AssignAttorneyModal';
 import React from 'react';
 import { ToggleModalButton } from '../lib/components/uswds/modal/ToggleModalButton';
 import Api from '@/lib/models/api';
 import { Attorney } from '@/lib/type-declarations/attorneys';
-import * as FeatureFlags from '@/lib/hooks/UseFeatureFlags';
 import { getFullName } from '@common/name-helper';
 
+const susan = new Attorney('Susan', 'Arbeit', 'Manhattan');
+const mark = new Attorney('Mark', 'Bruh', 'Manhattan');
+const shara = new Attorney('Shara', 'Cornell', 'Manhattan');
+const brian = new Attorney('Brian', 'Masumoto', 'Manhattan', { middleName: 'S' });
+const joe = new Attorney('Joe', 'Cornell', 'Manhattan');
+const bob = new Attorney('Bob', 'Cornell', 'Manhattan');
+const frank = new Attorney('Frank', 'Cornell', 'Manhattan');
+const sally = new Attorney('Sally', 'Cornell', 'Manhattan');
+const may = new Attorney('May', 'Cornell', 'Manhattan');
```

Comment:
These are real attorneys. Unless a test is getting attorneys from `attorneys.mock.ts`, which is only in the backend.